src/apiv2.ts

```diff
@@ -408,6 +414,13 @@ export class Client {
   }
 }
 
+function isLocalInsecureRequest(urlPrefix: string): boolean {
+  const u = parse(urlPrefix);
+  const isLocal = u.hostname === "localhost" || u.hostname === "127.0.0.1";
```

Comment:
You can use emulators on a different host too, like `FIRESTORE_EMULATOR_HOST=192.168.1.0 firebase ... `. How shall we handle that? I'd propose dropping `isLocal` check and just don't send real tokens over any insecure connection, like Firestore Admin SDK.

Reply:
+1, `0.0.0.0` is very common as are the examples that Yuchen mentioned.  Sending secure tokens over `http` is always a bad idea and we should drop them.

Reply:
Alright. It has been simplified.